airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/SnowflakeDestination.java

```diff
@@ -53,4 +53,18 @@ public SerializedAirbyteMessageConsumer getSerializedMessageConsumer(final JsonN
 
   }
 
+  public static boolean useAsyncSnowflake(final JsonNode config) {
+    // this is how we toggle async snowflake on.
+    // final Set<String> stagingLoadingMethods = Collections.emptySet();
+    final Set<String> stagingLoadingMethods = Set.of("standard", "internal staging", "internal-staging", "internal_staging");
```

Comment:
@davinchia Should this be on? In the other PR this was set to off

Reply:
hmm which pr did you previously see this from?

Reply:
note: from [the spec.json](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/destination-snowflake/src/main/resources/spec.json#L192-L194), I believe this should only be `Internal Staging`. We don't want to turn this on for 'standard'.